src/modules/polls.ts

```diff
@@ -279,18 +277,18 @@ export async function handleCommand(
 		}
 		case "list": {
 			try {
-				const polls = await prisma.attendancePoll.findMany();
+				const polls = await prisma.poll.findMany();
 				await interaction.editReply({
 					embeds: [
 						new MessageEmbed()
-							.setTitle("Attendance Polls")
+							.setTitle("Polls")
 							.setDescription(
 								polls.length
-									? "Below is a list of all attendance polls with their title and ID"
-									: "No attendance polls found"
+									? "Below is a list of all polls with their title and ID"
+									: "No polls found"
 							)
 							.addFields(
-								polls.map((p) => ({
+								polls.map((p: Poll) => ({
```

Comment:
If `polls` has type `Poll[]`, TS can infer that `p` has type `Poll`; why the annotation? I get no error without this
```suggestion
								polls.map((p) => ({
```

Reply:
I believe I wrote that when i was studying the code at first. It can be removed